.github/workflows/pr-test.yml

```diff
@@ -1,9 +1,19 @@
 name: "PR test"
 on: pull_request
 
+concurrency:
+  group: ${{ github.event.number || github.run_id }}
+  cancel-in-progress: true
+
 jobs:
   unit-test-with-asan:
     runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        arch:
+          - x86_64
+          - aarch64
```

Comment:
Actually I'm not sure if it's really needed to run the unittest on multiple archs.

Reply:
Could please check the actual result?  It seems it didn't run the test on an ARM (64-bit) machine.

Reply:
will check it today,  sorry for the inconvenience